docs/java-rest/high-level/cluster/health.asciidoc

```diff
@@ -137,6 +137,7 @@ include-tagged::{doc-tests-file}[{api}-response-nodes]
 --------------------------------------------------
 <1> Number of nodes in the cluster
 <2> Number of data nodes in the cluster
+<3> Whether cluster has voting exclusions
```

Comment:
Could `voting exclusions` here be a link to the section entitled `Voting configuration exclusions API`?

Reply:
https://github.com/elastic/elasticsearch/pull/38568/commits/0c28c0f5cf312a835843918a11a03b42102c1dce